content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -526,6 +526,6 @@ When you enable the API Priority and Fairness feature, the kube-apiserver serves
 
 
 For background information on design details for API priority and fairness, see
-the [enhancement proposal](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md).
+the [enhancement proposal](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1040-priority-and-fairness).
```

Comment:
Hi @amuraru . Thanks for the PR.

@wojtek-t ,
Would you confirm that this KEP file is the correct version? Thanks.

Reply:
Yes - thanks!